teammapper-backend/src/map/services/maps.service.spec.ts

```diff
@@ -88,6 +89,51 @@ describe('MapsController', () => {
 
       expect(createdNode).not.toBeUndefined()
     })
+
+    it('catches an FK error when trying to assign a nodeParentId to a root node', async () => {
```

Comment:
i am not sure I get this test. an empty map is added another node - why should there be an exception?

Reply:
You get a foreign key violation error if you try to add a nodeParentId to a root node, since a root node is not allowed to have a parent. This is caught by `maps.service.ts` but also has an extra test now. In practice, this test literally just covers line 65 and 66 in `maps.service.ts`: https://github.com/b310-digital/teammapper/pull/494/files#diff-bca4704425d3db9f75372bc5c102212b572d4202c46dea8b95d038341b5c7fa8R66